src/directions/main.ts

```diff
@@ -488,11 +521,28 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
 
       this.hoverpoint.geometry.coordinates = [e.lngLat.lng, e.lngLat.lat];
     }
-
+    this.currentMousePosition = e.point;
     this.draw();
+
+    /*
+     * if the user selected a waypoint or routeLine and routes should update while dragging,
+     * we initiate the live updating process.
+     */
+    if (this.configuration.refreshOnMove && !this.refreshOnMoveIsRefreshing) {
+      this.liveRefreshHandler(e);
+    }
   }
 
+  protected mouseMovementDetector: ReturnType<typeof setTimeout>;
```

Comment:
I'm not OK with the name. Sounds like some helper-method. Maybe something like `noMouseMovementTimer`? Or smtn similar...

Reply:
Changed